src/hooks/useDatatype.ts

```diff
@@ -0,0 +1,19 @@
+import useSWRImmutable from 'swr/immutable';
+import { openmrsFetch, OpenmrsResource } from '@openmrs/esm-framework';
+
+const conceptRepresentation =
+  'custom:(uuid,display,datatype,conceptMappings:(conceptReferenceTerm:(conceptSource:(name),code)))';
+
+export function useDatatype(references: Set<string>) {
```

Comment:
Assuming this returns concepts, what stops you from reusing the `useConcepts()` hook?

Reply:
I think we had this conversation a while back to create a new custom hook that also returns a filtered set with unresolved concepts, but if need be I can revert back to using the useConcept hook and modifying it.